src/api/control/async/asyncLinearMotionProfileController.cpp

```diff
@@ -213,12 +213,13 @@ void AsyncLinearMotionProfileController::executeSinglePath(const TrajectoryPair
   const auto reversed = direction.load(std::memory_order_acquire);
 
   for (int i = 0; i < path.length && !isDisabled(); ++i) {
+    float segDT = path.segment[i].dt;
```

Comment:
I probably should have done this to begin with...

Reply:
After looking at this again, I think it would be better to have
```suggestion
    const auto segDT = path.segment[i].dt * second;
```

The reason being that this method leaves no room for error in interpreting the units of `path.segment[i].dt`. Same change for below.